packages/flutter_tools/test/general.shard/ios/xcodeproj_test.dart

```diff
@@ -676,10 +678,189 @@ Information about project "Runner":
       fs.file(xcodebuild).createSync(recursive: true);
     });
 
+    group('arm simulator', () {
+      FakeProcessManager fakeProcessManager;
+      XcodeProjectInterpreter xcodeProjectInterpreter;
+
+      setUp(() {
+        fakeProcessManager = FakeProcessManager.empty();
+        xcodeProjectInterpreter = XcodeProjectInterpreter.test(processManager: fakeProcessManager);
+      });
+
+      testUsingContext('does not arm64 simulator when supported by all plugins', () async {
```

Comment:
This test title accidentally the verb :)

Reply:
Eagle eye.